components/org.wso2.finance.open.banking.conformance.mgt/src/main/java/org/wso2/finance/open/banking/conformance/mgt/dao/ReportDAO.java

```diff
@@ -0,0 +1,16 @@
+package org.wso2.finance.open.banking.conformance.mgt.dao;
```

Comment:
add license header to all the files(there are multiple files without the header)

Reply:
done